client/stripe-utils/utils.js

```diff
@@ -16,60 +16,13 @@ import { errorTypes, errorCodes } from './constants';
  * @return  {StripeServerData} Stripe server data.
  */
 const getStripeServerData = () => {
-	// Classic checkout or blocks-based one.
-	const stripeServerData =
-		typeof wc_stripe_upe_params !== 'undefined' // eslint-disable-line camelcase
-			? wc_stripe_upe_params // eslint-disable-line camelcase
-			: wc?.wcSettings?.getSetting( 'stripe_data', null );
-	if ( ! stripeServerData ) {
+	// Classic checkout.
+	// eslint-disable-next-line camelcase
+	if ( ! wc_stripe_upe_params ) {
 		throw new Error( 'Stripe initialization data is not available' );
```

Comment:
Reverting this solves the `wc_stripe_upe_params is not defined` error
<img width="658" alt="Screen Shot 2021-09-22 at 17 12 52" src="https://user-images.githubusercontent.com/407542/134418373-08328fe1-5b64-43f6-aa66-573dc507b25e.png">
: